client/src/components/employer-component/employer-information-card/index.jsx

```diff
@@ -0,0 +1,264 @@
+
+  const [confirmEditDialog, setConfirmEditDialog] = useState(false);
+  const [confirmCancelEditDialog, setConfirmCancelEditDialog] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorObj, setErrorObj] = useState(null);
+
+  const toggleEditable = () => {
+    if (editable) {
+      setConfirmCancelEditDialog(true);
+      return;
+    }
+    setEditable(!editable);
+  };
+
+  const phoneNumberWrapper = (phoneNumber) => {
+    return <a href={`tel:${phoneNumber}`}>{phoneNumber}</a>;
+  };
+
+  const faxWrapper = (fax) => {
+    return <a href={`tel:${fax}`}>{fax}</a>;
+  };
+
+  const emailWrapper = (email) => {
+    return <a href={`mailto:${email}`}>{email}</a>;
+  };
+
+  const websiteWrapper = (website) => {
+    return <a href={website}>{website}</a>;
+  };
+
+  const commitEdit = (e) => {
+    e.preventDefault();
+    setConfirmEditDialog(true);
+  };
+
+  const cancelEdit = () => {
+    setConfirmEditDialog(false);
+  };
+
+  const cancelEditUnsaved = () => {
+    setConfirmCancelEditDialog(false);
+  };
+
+  const handleConfirmCancel = () => {
+    setConfirmCancelEditDialog(false);
+    setEditable(false);
+    setEmployerName(employer.name);
+    setEmployerPhoneNumber(employer.phone_number);
+    setEmployerFax(employer.fax);
+    setEmployerEmail(employer.email);
+    setEmployerWebsite(employer.website);
+    setEmployerNAICSCode(employer.naics_code);
+    setEmployerAddress(employer.address);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    const modifiedEmployerInfo = {
+      id: employer.id,
+      name: employerName,
+      phone_number: employerPhoneNumber,
+      fax: employerFax,
+      email: employerEmail,
+      website: employerWebsite,
+      naics_code: employerNAICSCode,
+      address: employerAddress,
+    };
+
+    try {
+      const response = await modifyEmployerInfo(modifiedEmployerInfo);
+      console.warn(employer);
+      console.warn(response);
+      if (response.ok) {
+        setSnackBarMessage("Job lead updated successfully.");
+        setEditable(false);
+      } else {
+        setSnackBarMessage("Failed to update job lead.");
+      }
+    } catch (error) {
+      setErrorObj(error);
+      setSnackBarMessage("An error occurred.");
+    } finally {
+      setIsLoading(false);
+      setConfirmEditDialog(false);
+    }
+  };
+
+  // on initial startup, since useState() does not work on passed in props
+  useEffect(() => {
+    setEmployerName(employer.name);
+    setEmployerPhoneNumber(employer.phone_number);
+    setEmployerFax(employer.fax);
+    setEmployerEmail(employer.email);
+    setEmployerWebsite(employer.website);
+    setEmployerNAICSCode(employer.naics_code);
+    setEmployerAddress(employer.address);
+  }, [employer]);
+
+  if (errorObj) return <ErrorScreenComponent message={errorObj.message} />;
+
+  return (
+    <Card
```

Comment:
Same as the contact one.

Take a look at the edit-job-lead-form component seeing how the dividers are set up for the card.

This could clean up some of the spacing in the header of the card and center the text a bit more

Using Grid could simplify the handling of the rows https://mui.com/material-ui/react-grid/